packages/terra-button-group/src/ButtonGroupButton.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { PropTypes } from 'react';
```

Comment:
My main concern with that is that there are some button styles that we don't want the consumers to be able to set on a per button basis, but rather be set on the button group as a whole (size, isCompact, variant). We could pass those props down from the ButtonGroup to the ButtonGroup.Button to be set on Button, but those shouldn't be exposed to the consumer in the api for the ButtonGroup.Button.